test/python/quantum_info/operators/symplectic/test_clifford.py

```diff
@@ -586,7 +586,7 @@ def _define(self):
                 qc.sdg(0)
                 self.definition = qc
 
-        Clifford(MyHGate())
+        print(Clifford(MyHGate()))
```

Comment:
This looks like an unintentional debugging change.

Reply:
Yes, I'll remove it.